com/expectinit.exp

```diff
@@ -0,0 +1,54 @@
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# expectinit.exp is meant to be sourced from all test framework expect scripts in order
+# to set up each expect environment uniformly. This way test output does not depend upon
+# what the shell is like where the expect is called from.
+#
+#	USAGE:  source from within your expect script AFTER spawning your expect scripts shell
+#
```

Comment:
Add a note here that the shell prompt is reset to SHELL and so that is what the parent script should be looking for if/when they expect the shell prompt.